default-plugins/tab-bar/src/tab.rs

```diff
@@ -47,14 +47,15 @@ pub fn tab_style(
     is_sync_panes_active: bool,
 ) -> LinePart {
     let sync_text = match is_sync_panes_active {
-        true => " (Sync)".to_string(),
-        false => "".to_string(),
+        true => " (Sync)",
+        false => "",
     };
-    let tab_text = if text.is_empty() {
-        format!("Tab #{}{}", position + 1, sync_text)
+    let mut tab_text = if text.is_empty() {
+        format!("Tab #{}", position + 1)
     } else {
         text
     };
+    tab_text.push_str(sync_text);
```

Comment:
As a matter of style and efficiency, I'd probably just write:

```rust
if is_sync_panes_active {
    tab_text.push_str(" (Sync)");
}
```

But I'm happy to merge either!

Reply:
Done.